packages/flutter/lib/src/widgets/service_extensions.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
These consts all get exported into the global namespace for users of the Flutter framework. Can we namespace them better to keep them somewhat more hidden?

Reply:
Created a file `flutter/lib/service_extensions.dart` which will export this file, and I removed the export from `flutter/lib/widgets.dart`